.github/workflows/openapi-diff.yml

```diff
@@ -47,7 +40,7 @@ jobs:
           ref: 'main'
           repository: getsentry/sentry-api-schema
           path: sentry-api-schema
-          token: ${{ steps.getsentry.outputs.token }}
+          token: ${{ github.token }}
```

Comment:
do you know if you need a token at all since it's public?

Reply:
i'll try removing it

Reply:
yup, it worked 👍🏼